requirements.txt

```diff
@@ -1,2 +1,2 @@
 websockets>=10.0
-aiohttp>=3.7.4post0,<3.8.0
+aiohttp>=3.7.4post0,<4.0.0
```

Comment:
Do pincer has been tested on 4.0.0 of aiohttp?

Reply:
accidently bumped but no and it doesn't install 4.0, will undo because the new pull should do this.